### PR TITLE
Implemented basic functionality for /Home

### PR DESCRIPTION
**Pull request by ndubs18**
added components Home, Coinlist and Coin and an asynchronous call to the coingecko /markets endpoint. Basic coin information is rendered in the home page/component.

### CODE REVIEW

**Comment by ethanSE** [comment]
Cool I'm taking a look now

File: client/src/App.js [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
+import 'bootstrap/dist/css/bootstrap.min.css';
+
 // We import all the components we need in our app
 import Navbar from "./components/navbar";
 import RecordList from "./components/recordList";
 import Edit from "./components/edit";
 import Create from "./components/create";
 
+//view components
+import Home from "./components/welcome";
```
Comment (ethanSE) [inline client/src/App.js:16]: I'm getting an error here. Is there a file "./components/welcome" that was not commited?

Comment (ndubs18) [reply client/src/App.js:16]: Ah sorry I ended up changing a file/component name and forgot to change the path

File: client/src/App.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ const App = () => {
           .then(response => response.json())
           .then(data => {
```
Comment (ethanSE) [inline client/src/App.js:25]: Not suggesting a change, just background info

Whenever you have an array and want to produce a new array of the same length this is a good use case for Array.prototype.map(). 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

```javascript
.then((data) => {
        const coinList = data.map((coin) => ({
          id: coin.id,
          name: coin.name,
          symbol: coin.symbol,
          price: coin.current_price,
          priceChange: coin.price_change_percentage_24h,
          image: coin.image,
        }));
        setCoinData(coinList);
      });
```

more background info:

"map" is a common name for the pattern of creating a new collection of values with the same structure as the original collection by applying a transformation function to each value.

https://en.wikipedia.org/wiki/Map_(higher-order_function).

I the case of javascript arrays this means a new array with the same length, but with the contents transformed by the provided callback function.

**Review by ethanSE — APPROVED** [review]
Looks good to me

**Pull request merged**